scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -682,7 +682,9 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
              */
             $scope.close = function() {
                 _closing = true;
-                authoring.close($scope.item, $scope.origItem, $scope.save_enabled()).then(() => {
+
+                // returned promise used by superdesk-fi
```

Comment:
We could make a simple test just to check if this is returning a promise (or just checking it works properly) instead of writing a comment

Reply:
I'm adding a comment, because of external usage. Someone would see that the code is not used in superdesk-client-core and might remove it. I'd have to add the same comment in the test case, so I might as well keep it here.